dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
This file needs to go into the _**.github**_ folder, not the **.github/workflows/** folder.

Reply:
It is in the .github folder.

Reply:
I tried to do move it there at the time and it didn't allow me, so I thought that's where it was supposed to go.